### PR TITLE
raised to jdk6 (#196)

### PR DESCRIPTION
**Pull request by lkwg82**
#196

### CODE REVIEW

File: pitest-maven-verification/pom.xml [hunk 1]
```diff
@@ -32,13 +32,6 @@
 			<plugin>
 				<artifactId>maven-failsafe-plugin</artifactId>
 				<version>2.12</version>
-				<dependencies>
-					<dependency>
-						<groupId>org.apache.maven.surefire</groupId>
-						<artifactId>surefire-junit47</artifactId>
-						<version>2.12</version>
```
Comment (szpak) [inline pitest-maven-verification/pom.xml]: I wonder why it was removed? surefire-junit47 became a default dependency in the current surefire (I could miss it as I  migrated to Gradle some time ago) or no features from surefire-junit47 are actually used in PIT?

Comment (szpak) [inline pitest-maven-verification/pom.xml]: I've just found the previous commit "try to fix hanging build for jdk7". Did it help?

**Comment by lkwg82** [comment]
I dont think it helped. It helped like clicking retry. ;)

**Comment by lkwg82** [comment]
Is it already done and because of that closed and not merged?

**Comment by hcoles** [comment]
Sorry, should have added a comment when closing - yes the builds been separately updated to 1.6. Forgot this PR was here.

**Comment by lkwg82** [comment]
No problem. Great!

**Pull request closed without merging**